CHANGELOG.md

```diff
@@ -3,6 +3,10 @@ All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 Use the following format for additions: ` - VERSION: [feature/patch (if applicable)] Short description of change. Links to relevant issues/PRs.`
 
+- 1.4.33:
+  - Bump getmac version [#1130](https://github.com/FredrikNoren/ungit/issues/1130)
+  - Add config to disable animation [#1136](https://github.com/FredrikNoren/ungit/issues/1136)
+  - Remove node6. Add node9 and node 10.
```

Comment:
Wasn't node 10 already there because `"node"` uses the latest stable release automatically?
Looks more like "Add node 8 and 9" to me.